app/workers/va_notify_dd_email_job.rb

```diff
@@ -23,7 +23,7 @@ def self.send_to_emails(user_emails, dd_type)
   def perform(email, dd_type)
     notify_client = VaNotify::Service.new(Settings.vanotify.services.va_gov.api_key)
     template_type = "direct_deposit_#{dd_type.to_sym == :ch33 ? 'edu' : 'comp_pen'}"
-    template_id = Settings.vanotify.template_id.public_send(template_type)
+    template_id = Settings.vanotify.services.va_gov.template_id.public_send(template_type)
```

Comment:
That change requires environment specific settings in devops repository to be merged first.
Has this been done?

Reply:
Yes, Phil has done it.

Reply:
Can you link to that devops PR in this PR's description for easier traceability?

Reply:
Done